src/components/ToggleButton/ToggleButton.js

```diff
@@ -57,6 +57,24 @@ export default class ToggleButton extends Component {
     );
   }
 
+  renderGridList(classes, value, title) {
+    const joinedClasses = classNames(classes, 'slider-teal');
+
+    return (
+      <div className="slider--grid-list">
+        <FontAwesomeIcon
+          icon={['fas', 'list']}
+          className="slider--grid-list--icon"
```

Comment:
This should be `slider-icon--grid-list` - so there is `slider` and its modifiers, `slider-icon` and its modifiers